### ISSUE TITLE
Keyboard focus returns to the "Select Subscription" combo box, but NVDA announces the name of the "select subscription" list and the currently focused list item: A11y_Azure Machine Learning Extension for VS Code_Appling filter on subscription_Screen reader

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode-tools-for-ai#2491]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev2; #BM_Azure Machine Learning Extension for VS Code_Win32_Jan2024; #Azure Machine Learning Extension for VS Code; #WCAG1.3.1; #FTP; #Screen Reader; #AILimited; #NVDA; #JAWS; #Element:Combobox;

## Environment Details:
Application: Azure Machine Learning Extension for VS Code
Version: 1.2.0
OS: Windows 11 Enterprise 24H2
Build: 26100.2605
Screen reader
JAWS Version: 2024
NVDA Version: 2024.2.0

## Repro Steps

1. Tun on Jaws screen reader
2. Launch the VS Code.
3. Install the Azure Machine Learning extension and sign in with appropriate Credentials.
4. Tab until the left toolbar (list of icons). This toolbar has a file icon, search icon, settings, etc.
5. Use the up/down arrow keys to navigate to the "Azure" icon and press enter
6. The Azure panel will now be active next to the toolbar.
7. Tab until the Machine Learning dropdown.
8. Tab till any subscription for example AML V1 Personal subscription.
9. Tab until the "Filter" button and press Enter. It will be a button with a funnel icon.
10. TAB till "Select subscriptions" combo box.
11. TAB till "Select subscriptions" list and press down arrow key to move focus to any list item.
12. Press "Shift + TAB" key to navigate "Select subscriptions" combo box.
13. Observer that NVDA announces the name of the "select subscription" list and the currently focused list item.

## Actual Experience
When the keyboard focus returns to the "Select Subscription" combo box using the Shift + Tab key, NVDA announces the name of the "select subscription" list and the currently focused list item.

### Similar issue observed with following scenarios:

1. **Scenario1**: Creating a job Select an Azure ML job Schema
2. TAB till create job button and press ENTER key
3. TAB till list and observe the issue.
4. **Scenario2**: Setting Default Workspace
5. Open the Command Palette via the keyboard command Ctrl + Shift + P 
6. Type "Set default workspace" and select the option from the dropdown list called "Azure ML: Set Default Workspace") and press enter
7. A list should appear that says "Select a subscription"
8. Use the up/down arrows to navigate the list until "AML V1 Personal 2" is selected, and press enter
9. A list should appear that says "Select a workspace"
10. TAB till list and observe the issue.

## Expected Experience
When the keyboard focus returns to the "Select Subscription" combo box using the Shift + Tab key, NVDA announces about the "Select Subscription" combo box only. 

## User Impact:
Screen reader user will miss lead as focus lands on "Select Subscription" combo box bug screen reader announcing about the name of the "select subscription" list and the currently focused list item.

**Comment by JerryYangKai** [comment]
Hello @kapilvaishna  
I think this experience is by design from VS Code side, All the VS Code extension need to follow this UI guideline and cannot change it. https://vscode-api.js.org/interfaces/vscode.QuickInput.html

This is what I tried with another Microsoft official extension, the Azure Resources, the screenshot is 
 
![Image](https://github.com/user-attachments/assets/376d6a44-2635-4687-802e-649d18c2c858)

You can find that in Azure Resources it will also both announce the tittle and the first selection.

**Comment by kupatkar99** [comment]
GithubTags:#ResolvedByTT
